content/docs/getting-started/futures.md

```diff
@@ -205,6 +205,8 @@ use std::time::Duration;
 
 use futures::Future;
 use futures_cpupool::CpuPool;
+// There's also a tokio_core::reactor::Timeout, but it needs an event loop
+// to run on.
```

Comment:
Perhaps this could be delayed to later or explained elsewhere? I find that comments like this unfortunately detract from the example at hand much of the time.